src/averageAges.js

```diff
@@ -15,6 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let men = people.filter(person => person.sex === 'm');
+
+  century && (men = men
+    .filter(person => Math.ceil(person.died / 100) === century));
+
+  const sumAge = men.reduce(
+    (sum, man) => sum + (man.died - man.born), 0
+  );
+
+  return sumAge / men.length;
```

Comment:
move it to the separate function

Reply:
why is it bad to use it inside reduce ?

Reply:
you should move reduce, because you use logic of getting average age twice.
If you have any questions about mentor comments - you can ask in in the fe_chat, instead of requesting re-review